### DISCUSSION TITLE
Difference between Binaryify's version of One Dark and your's?

### DISCUSSION THREAD

**Discussion by davisssamuel** [one-dark/vscode-one-dark-theme#43] [Q&A]
The first impression I had was that your version uses italic text more and, unlike Binaryify's version, doesn't use red text for variables, which I like more because red is associated with errors, at least in my head.

**Comment by mskelton** [comment]
Is there an issue or is this just a discussion?  Trying to discern from your description which doesn't seem to mention any issues, so I'm inclined to move this to discussions (which I just enabled).

**Comment by davisssamuel** [comment]
No issue, just curious

**Comment by mskelton** [comment]
The two things you mentioned are the biggest changes, but I also wrote this theme from scratch to attempt to bring more consistency between languages as the original version by [Binaryify](https://github.com/Binaryify) over time gained a lot of inconsistencies.
